src/components/Gradebook/index.jsx

```diff
@@ -357,6 +343,7 @@ export default class Gradebook extends React.Component {
               onClose={() => this.props.updateBanner(false)}
               open={this.props.showSuccess}
             />
+            {PageButtons(this.props)}
```

Comment:
The way data passed to components here can be improved. @reillz10 please help out here.